tests/detect-pcrexform-01/test.yaml

```diff
@@ -0,0 +1,8 @@
+requires:
+
```

Comment:
Should we set a min-version of 6.0 ?

Reply:
Thanks for pointing this out. I'll update the `requires` section